addons/osfstorage/tests/test_views.py

```diff
@@ -215,6 +215,17 @@ def test_metadata_not_found(self):
         assert_equal(res.status_code, 404)
 
     def test_metadata_not_found_lots_of_slashes(self):
+        res = self.send_hook(
```

Comment:
I think this new behavior is more consistent because the node permissions determine permissions, not the file path,